### PR TITLE
Update tenant theme doc

### PR DESCRIPTION
**Pull request by chathuranga95**
### Description

This PR updates the screenshots and instructions to follow when uploading/downloading the devportal tenant theme from the APIM 3.2.0 admin portal.

### CODE REVIEW

File: en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ Ex: Enable landing page. **defaultTheme.json** ( Teant theming ).
 ```
 ### Tenant Theming 
 
-#### Uploading Devportal theme via the Admin Portal (Tenants Only)
+#### Uploading/Downloading Devportal theme via the Admin Portal (Tenants Only)
```
Comment (shaniR) [inline en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md:42]: Do we need to specifically mention about download?

File: en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md [hunk 2]
```diff
@@ -69,16 +69,20 @@ Ex: Enable landing page. **defaultTheme.json** ( Teant theming ).
...
+    Alternatively, you can drag and drop your ZIP file to upload. 
 
     ![Upload tenant theme]({{base_path}}/assets/img/learn/upload-tenant-theme.png)
 
-5.  Access the API Developer Portal (`https://<server-host>:9443/devportal`) using your tenant username and password.
+5.  Access the API Developer Portal (`https://<server-host>:<server-port>/devportal`) using your tenant username and password.
 
     Note the new theme is applied.
     
+!!!Note
```
Comment (shaniR) [inline en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md:83]: If the UI is self explanatory, there's no point mentioning this in the docs.

File: en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md (hunk 1 above)
Comment (chathuranga95) [reply en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md]: Fixed with: https://github.com/wso2/docs-apim/pull/1659/commits/04ddc59e90b732a370a7e4752bbbc7ac16f267f0

File: en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md (hunk 2 above)
Comment (chathuranga95) [reply en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md]: Fixed with: https://github.com/wso2/docs-apim/pull/1659/commits/04ddc59e90b732a370a7e4752bbbc7ac16f267f0

**Pull request merged**